python/setup.py

```diff
@@ -168,12 +25,16 @@ def finalize_options(self):
         "Programming Language :: Python :: 3.9",
     ],
     # Include the separately-compiled shared library
-    setup_requires=["Cython>=0.29,<0.30"],
     extras_require={"test": ["pytest", "pytest-xdist"]},
-    ext_modules=extensions,
     packages=find_packages(exclude=["tests*"]),
-    package_data={"": ["*.pyi"]},
-    cmdclass=cmdclass,
-    install_requires=install_requires,
+    package_data={
+        # Note: A dict comprehension with an explicit copy is necessary (rather
+        # than something simpler like a dict.fromkeys) because otherwise every
+        # package will refer to the same list and skbuild modifies it in place.
+        key: ["*.pyi", "*.pxd"]
+        for key in find_packages(include=["kvikio._lib"])
+    },
+    cmdclass=versioneer.get_cmdclass(),
+    install_requires=["Cython>=0.29,<0.30"],
```

Comment:
Is this just how `scikit-build` handles build dependencies?

Also is this still needed given it is in `pyproject.toml` above?

Reply:
What do you mean by "this"?

No we don't need this. I just copied it in without thinking. Cython is only a build-time requirement, not an install-time requirement, so it should only have been in pyproject.toml to begin with.

Reply:
Sorry by this just meant using `install_requires` to handle build dependencies. Didn't know if `scikit-build` was reinterpreting `install_requires` in some way. In any event sound like no and it has since been dropped.